run/namelist.figureCD.cpu

```diff
@@ -0,0 +1,363 @@
+
+ &param0
+ nx           =     256,
+ ny           =     256,
+ nz           =     256,
+ ppnode       =      128,
+ timeformat   =       2,
+ timestats    =       1,
+ terrain_flag = .false.,
+ procfiles    = .false.,
+ /
+
+ &param1
+ dx     =    20.0,
+ dy     =    20.0,
+ dz     =    10.0,
+ dtl    =   0.500,
+ timax  = 60.0,
+ run_time =  -999.9,
+ tapfrq =   900.0,
+ rstfrq = 10800.0,
+ statfrq =  60.0,
+ prclfrq =   60.0,
+ /
+
+ &param2
+ cm1setup  =  1,
+ testcase  = 10,
+ adapt_dt  =  1,
+ irst      =  0,
+ rstnum    =  1,
+ iconly    =  0,
+ hadvordrs =  5,
+ vadvordrs =  5,
+ hadvordrv =  5,
+ vadvordrv =  5,
+ advwenos  =  2,
+ advwenov  =  0,
+ weno_order = 5,
+ apmasscon =  0,
+ idiff     =  0,
+ mdiff     =  0,
+ difforder =  6,
+ imoist    =  1,
+ ipbl      =  0,
+ sgsmodel  =  4,
+ tconfig   =  1,
+ bcturbs   =  1,
+ horizturb =  0,
+ doimpl    =  0,
+ irdamp    =  1,
+ hrdamp    =  0,
+ psolver   =  2,
+ ptype     =  0,
+ ihail     =  0,
+ iautoc    =  0,
+ icor      =  1,
+ lspgrad   =  3,
+ eqtset    =  1,
+ idiss     =  0,
+ efall     =  0,
+ rterm     =  0,
+ wbc       =  1,
+ ebc       =  1,
+ sbc       =  1,
+ nbc       =  1,
+ bbc       =  3,
+ tbc       =  1,
+ irbc      =  4,
+ roflux    =  0,
+ nudgeobc  =  0,
+ isnd      =  8,
+ iwnd      =  8,
+ itern     =  0,
+ iinit     =  0,
+ irandp    =  1,
+ ibalance  =  0,
+ iorigin   =  1,
+ axisymm   =  0,
+ imove     =  1,
+ iptra     =  0,
+ npt       =  1,
+ pdtra     =  1,
+ iprcl     =  1,
+ nparcels  =  8000000,
```

Comment:
Your choice.  I would encourage you to collect data for different values and see if any interesting story emerges.  I selected 8M on 8 nodes, because the cost to run can be pretty large if it is increase any more.